### PR TITLE
docs(en-US): references -> user profile -> basic and custom data

### PR DESCRIPTION
**Pull request by IceHe**
References -> user profile -> basic data, custom data, and identities

image

### CODE REVIEW

File: docs/docs/references/users/README.md [hunk 1]
```diff
@@ -11,28 +11,28 @@ A user profile consists of the following types of data:
 
 - [**Basic data**](#basic-data): contains basic info in pre-defined attributes, such as user id, username, email, phone number, and when the user last signed in.
 - [**Custom data**](#custom-data): contains additional info in customizable attributes, such as user-preferred color and language.
-- [**Identities**](#identities): contains the user info retrieved from social sign-in, such as Facebook, GitHub, and WeChat.
+- [**Identities**](#identities): contains the user info retrieved from social sign-in (a.k.a. sign-in with a social connector), such as Facebook, GitHub, and WeChat.
```
Comment (IceHe) [inline docs/docs/references/users/README.md:14]:
```suggestion
- [**Identities**](#identities): contains the user info retrieved from social sign-in (i.e., sign-in with a social connector), such as Facebook, GitHub, and WeChat.
```

File: docs/docs/recipes/manage-users/using-management-api.md [hunk 2]
```diff
@@ -14,15 +14,15 @@ The user-related RESTful APIs are mounted at `/api/users` except the user activi
...
-| [Remove social connection](../../../docs/recipes/manage-users/using-admin-console#view-and-update-user-profile) | [`DELETE /api/users/:userId/identities/:target`](/api/#tag/Users/paths/~1api~1users~1:userId~1identities~1:target/delete) |
+| Admin Console - User Management                                                                                 | Management API                                                                                                                                       |
+| :-------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [List users](../../../docs/recipes/manage-users/using-admin-console#list-and-search-users)                      | <a href="/api/#tag/Users/paths/~1api~1users/get" target="_blank"> `GET /api/users`</a>                                                               |
```
Comment (wangsijie) [inline docs/docs/recipes/manage-users/using-management-api.md:19]: is it necessary?

File: docs/docs/references/users/README.md [hunk 3]
```diff
@@ -80,10 +80,137 @@ A user that signed in with Facebook:
...
+
+_Name_ is the user's full name.
+It should be at most 128 characters.
+
+#### Avatar
+
+_Avatar_ is the URL pointing to the user's avatar image.
+
+If the user registers with a social connector like Facebook and WeChat, their _avatar_ may be retrieved from the social user info.
+It should be at most 2048 characters.
```
Comment (wangsijie) [inline docs/docs/references/users/README.md:126]: nit: How about saying "the max-length is xxx" instead of "it should be"

**Review by wangsijie — APPROVED** [review]
Overall LGTM

File: docs/docs/references/users/README.md (hunk 3 above)
Comment (IceHe) [inline docs/docs/references/users/README.md:126]:
```suggestion
Its max length is 2048.
```

Comment (IceHe) [reply docs/docs/references/users/README.md:126]: Updated to 
```suggestion
Its max length is 2048.
```

File: docs/docs/references/users/README.md [hunk 4]
```diff
@@ -80,10 +80,137 @@ A user that signed in with Facebook:
...
+_Primary phone_ is the user's phone number, used for sign-in with the phone number and passcode from SMS.
+
+When the user has not registered with their phone number, their _primary phone_ may be empty.
+When the _primary phone_ is non-empty, it should contain numbers prefixed with the
+[country calling code](https://en.wikipedia.org/wiki/List_of_country_calling_codes) (excluding the plus sign `+`).
+
+#### Name
+
+_Name_ is the user's full name.
+It should be at most 128 characters.
```
Comment (IceHe) [inline docs/docs/references/users/README.md:119]:
```suggestion
Its max length is 128.
```

**Pull request merged**